app/src/main/kotlin/org/equeim/tremotesf/ui/addtorrent/AddTorrentDirectoriesAdapter.kt

```diff
@@ -29,19 +35,24 @@ class AddTorrentDirectoriesAdapter(
             "org.equeim.tremotesf.ui.addtorrent.AddTorrentDirectoriesAdapter.items"
     }
 
-    private val items: ArrayList<String>
+    private var items = ArrayList<String>()
```

Comment:
It looks like empty list initialization can be moved to the else branch in init {} (before launching coroutine). Otherwise we will just overwrite it when saved != null.

Reply:
I don't see how initializing it in else{} solves anything...